Description: wait a second after go serve socket
 Multiple upstream wait time by hundred.
Author: Laszlo Boszormenyi (GCS) <gcs@debian.org>
Last-Update: 2018-09-23

---

--- thrift-0.11.0.orig/lib/go/test/tests/multiplexed_protocol_test.go
+++ thrift-0.11.0/lib/go/test/tests/multiplexed_protocol_test.go
@@ -66,7 +66,7 @@ func TestMultiplexedProtocolFirst(t *tes
 
 	defer server.Stop()
 	go server.Serve()
-	time.Sleep(10 * time.Millisecond)
+	time.Sleep(1000 * time.Millisecond)
 
 	transport, err := createTransport(addr)
 	if err != nil {
@@ -105,7 +105,7 @@ func TestMultiplexedProtocolSecond(t *te
 
 	defer server.Stop()
 	go server.Serve()
-	time.Sleep(10 * time.Millisecond)
+	time.Sleep(1000 * time.Millisecond)
 
 	transport, err := createTransport(addr)
 	if err != nil {
